lambda/events_id/delete/index.js

```diff
@@ -1,96 +1,97 @@
 // aws-sdk is needed for its libraries that allow to access other aws services 
 const AWS = require('aws-sdk');
 // the DynamoDB.DocumentClient class allows access to Dynamodb directly 
-// region must be us-east-2 for Ohio
-const documentClient = new AWS.DynamoDB.DocumentClient({ region: 'us-east-2' });
-
+// region must be us-west-2 for Oregon/Prod
+const documentClient = new AWS.DynamoDB.DocumentClient({ region: 'us-east-2'}); 
```

Comment:
As noted in the comment this should be changed to us-west-2 for TravisCI to correctly push to prod.

Reply:
Thanks, I updated the region to us-west-2 for prod.

Reply:
Approve the change